VSRAD.Package/DebugVisualizer/VisualizerTable.cs

```diff
@@ -85,7 +86,8 @@ public VisualizerTable(ProjectOptions options, FontAndColorProvider fontAndColor
 
             _ = new ContextMenus.ContextMenuController(this, new ContextMenus.IContextMenu[]
             {
-                new ContextMenus.TypeContextMenu(this, VariableTypeChanged, AvgprStateChanged, ProcessCopy, InsertSeparatorRow),
+                new ContextMenus.TypeContextMenu(this, VariableTypeChanged, AvgprStateChanged, ProcessCopy, InsertSeparatorRow,
+                    AddWatchRange),
```

Comment:
`addWatchRange: (name, from, to) => ArrayRange.FormatArrayRangeWatch(name, from, to, options.VisualizerOptions.MatchBracketsOnAddToWatches).ToList().ForEach(AddWatch))`

Rationale:
* Passing `MatchBracketsOnAddToWatches` to `FormatArrayRangeWatch`
* Less code duplication

Reply:
Thanks, done in [c27480e](https://github.com/vsrad/radeon-asm-tools/pull/319/commits/c27480e0a708f94783bf026d34f57786f753a6a1)